diagnostic.md

```diff
@@ -45,7 +61,22 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+```CSS
+.big {
+
+}
 
+#contentPane {
+
+}
+
+.important, .red {
```

Comment:
This will target elements that have the `.important` class AND elements that have the `.red` class. We want only elements that have BOTH classes: `.important.red`